PharoTour/PharoTour.pillar

```diff
@@ -353,7 +352,7 @@ The ""System Browser"", also known as "Class Browser", is one of the key tools
 used for programming. As we shall see, there are several interesting browsers
 available for Pharo, but this is the basic one you will find in any image.
 
-@@todo TODO: factorial on number would be better. and should redo the screenshot
+@@authorToDo TODO: factorial on number would be better. and should redo the screenshot
```

Comment:
`@@authorToDo` is a note the author let in the text to remind every author or just himself to do something. I think both `@@authorToDo` and github issues have their uses so I would keep both. We could add a github issue to remind us to fix all the `@@authorToDo` before publication :-).